openpype/lib/events.py

```diff
@@ -348,14 +528,19 @@ def _process_event(self, event):
             event (Event): Prepared event with topic and data.
         """
 
-        invalid_callbacks = []
-        for callback in self._registered_callbacks:
+        def callback_sorter(item):
+            # Put callbacks without order to the end of the list
+            if item.order is None:
+                return (1, 0)
+            return (0, item.order)
```

Comment:
By doing this it does mean that now you can _never_ force methods either in front or behind those registered with a predefined order. Basically meaning that now with more lines of code, and also supporting `None` as an order (instead of a `0` default) we now support less flexibility and additionally need to document that `None` runs AFTER any numbered orders instead of just supporting numbers?

As such, I wonder whether the added complexity of allowing `None` really adds anything useful and thus warrant the additional code for it.

Thoughts @antirotor ?

I feel like usually adding an event in code without an explicit order means "run whenever" instead of "run last" (from the developer's point of view adding the callback). If now, after having five events that are running at unspecified orders I want to now run my additional added event absolutely last, no matter what - I can't do so by assigning an arbitrarily high integer, since I can't force it after any 'unsorted' ones.